moped-editor/src/views/projects/projectView/ProjectComponents/utils.js

```diff
@@ -0,0 +1,66 @@
+import { useMemo } from "react";
+import booleanIntersects from "@turf/boolean-intersects";
+import circle from "@turf/circle";
+
+/* Filters a feature collection down to one type of geometry */
+export const useFeatureTypes = (featureCollection, geomType) =>
+  useMemo(() => {
+    const features = featureCollection.features.filter((feature) => {
+      const thisGeom = feature.geometry.type.toLowerCase();
+      return thisGeom.includes(geomType);
+    });
+    return { type: "FeatureCollection", features };
+  }, [featureCollection, geomType]);
+
+/*
+Bit of a hack to generate intersection labels from nearby streets. 
+this relies on nearby lines being avaialble in-memory, which 
+is not guaranteed. a reliable solution would be query the AGOL streets
+layer on-the-fly to grab street names - not sure if this is worth it TBH 
+*/
+export const getIntersectionLabel = (point, lines) => {
+  var radius = 10;
+  var options = {
+    steps: 10,
+    units: "meters",
+  };
```

Comment:
I think you may want to use `radius` here instead of 10.

Reply:
I think this is ok but I think that @johnclary can speak to this better than me

Reply:
it's confusing to see `10` twice. the `steps` prop refers to the number of vertices in the circle. i clearly copied from [the example](https://turfjs.org/docs/#circle) as evidenced by the use of `var`.